backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/codec/impl/AbstractElasticsearchVectorFieldCodec.java

```diff
@@ -53,6 +53,11 @@ public F decode(JsonElement element) {
 		return fromJsonArray( JsonElementTypes.ARRAY.fromElement( element ) );
 	}
 
+	@Override
+	public F convertFromDoubleNullSafe(Double value) {
+		return null;
```

Comment:
It's unlikely that we'd want any aggregations for vectors, so maybe just an assertion error.